### PR TITLE
Typesafing PlotLineOrBand

### PR DESCRIPTION
**Pull request by TorsteinHonsi**

### CODE REVIEW

File: ts/Core/Axis/PlotLineOrBand/PlotLineOrBand.ts [hunk 1]
```diff
@@ -132,36 +142,21 @@ class PlotLineOrBand {
     public render(): (PlotLineOrBand|undefined) {
```
Comment (bre1470) [inline ts/Core/Axis/PlotLineOrBand/PlotLineOrBand.ts:142]: I do not see much value in this chain concept when `undefined` is involved.

File: ts/Core/Axis/PlotLineOrBand/PlotLineOrBand.ts [hunk 2]
```diff
@@ -219,68 +202,74 @@ class PlotLineOrBand {
...
-            path = axis.getPlotBandPath(from, to, options);
+            });
+        } else if (defined(from) && defined(to)) { // Plot band
+            path = axis.getPlotBandPath(
+                logarithmic?.log2lin(from) ?? from,
+                logarithmic?.log2lin(to) ?? to,
+                options
+            );
         } else {
             return;
```
Comment (bre1470) [inline ts/Core/Axis/PlotLineOrBand/PlotLineOrBand.ts:232]: Probably this return lacks the `this`.

**Review by bre1470 — APPROVED** [review]
Looks good! 👍

Some half-related comments...

File: ts/Core/Axis/PlotLineOrBand/PlotLineOrBand.ts (hunk 1 above)
Comment (TorsteinHonsi) [reply ts/Core/Axis/PlotLineOrBand/PlotLineOrBand.ts:142]: It seems that the use case is to return `undefined` when the options are not valid. I'll update the docs.

**Comment by TorsteinHonsi** [comment]
Pending checks

**Pull request merged**